### DISCUSSION TITLE
Library Manager

### DISCUSSION THREAD

**Discussion by Antonio-Raposo** [Sloeber/arduino-eclipse-plugin#1530] [Ideas]
How hard would it be to add a filter in Preferences -> Library Manager to show only the selected (aka downloaded) libraries?

**Comment by jantje** [comment]
I'm really bad at UI stuff so I don't know.
But I wanted that feature many times.

**Reply by Antonio-Raposo** [reply]
I pulled the sources and it was very easy to compile. I used the build script with cygwin (a linux shell in windows) and the only thing I needed to fix was the name from linux to w32. I will try to do something like that and if I can I will tell you.
